kratos/includes/kratos_parameters.h

```diff
@@ -287,6 +329,49 @@ class Parameters
         if(mpvalue->IsString() == false) KRATOS_THROW_ERROR(std::invalid_argument,"argument must be a string","");
         return mpvalue->GetString();
     }
+    Vector GetVector() const    
+    {
+        if(mpvalue->IsArray() == false) KRATOS_ERROR << "argument must be a Vector (a json list)" ;
+        
+        const unsigned int size = mpvalue->Size();
+        if(size < 2) KRATOS_ERROR << "argument is not a Vector, size < 2! " << std::endl;
```

Comment:
Well we could return a vector even of size 1. It is relevant in some corner cases